src/Service/NuRaftLogSnapshot.h

```diff
@@ -123,6 +166,12 @@ class KeeperSnapshotStore
         uint64_t & processed,
         uint32_t & checksum);
 
+    /// For async snapshot version v3
```

Comment:
Why is v3? Is there any modification of snapshot file format?

Reply:
> Why is v3? Is there any modification of snapshot file format?

Just follow https://github.com/JDRaftKeeper/RaftKeeper/blob/dbad184ff000eb451a3bce2728b1ce14b6a5b3b2/src/Service/NuRaftLogSnapshot.h#L98

Reply:
Please correct it to v2